.gitmodules

```diff
@@ -10,20 +10,12 @@
 #     rm -rf path_to_submodule
 
 
-[submodule "submodules/HaRe"]
-	path = submodules/HaRe
-	# url = https://github.com/bubba/HaRe.git
-	url = https://github.com/alanz/HaRe.git
-
 [submodule "submodules/cabal-helper"]
 	path = submodules/cabal-helper
-	# url = https://github.com/arbor/cabal-helper.git
-	url = https://github.com/alanz/cabal-helper.git
 	# url = https://github.com/DanielG/cabal-helper.git
+	# Change this back once https://github.com/DanielG/cabal-helper/pull/85/ merged
+	url = https://github.com/bubba/cabal-helper.git
```

Comment:
Are these still the right remotes?

Reply:
yeah.

Reply:
actually, we could look at DanielG c-h directly.